packages/data-point/lib/reducer-types/reducer-object/resolve.test.js

```diff
@@ -106,7 +108,16 @@ describe('resolve#reducerObject.resolve', () => {
     })
   })
 
-  it('should resolve a reducer object', () => {
+  it('should resolve a reducer object that returns one sepecfic key when root is passed as first elt, returns all keys when passed in last', () => {
```

Comment:
This is kinda... wordy? 😆  Can we make the explanation a bit more concise and clear?

I can't think of a way to describe it better... @acatl @raingerber?